src/elements/Order.php

```diff
@@ -181,12 +181,32 @@ protected static function defineSources(string $context = null): array
                 'criteria' => [],
                 'defaultSort' => ['dateOrdered', 'desc']
             ],
+            [
+                'key' => 'new',
+                'label' => Translations::$plugin->translator->translate('app', 'New'),
+                'criteria' => [
+                    'status' => [
+                        Constants::ORDER_STATUS_NEW
+                    ]
+                ],
+                'defaultSort' => ['dateOrdered', 'desc']
+            ],
             [
                 'key' => 'pending',
                 'label' => Translations::$plugin->translator->translate('app', 'Pending'),
                 'criteria' => [
                     'status' => [
-                        Constants::ORDER_STATUS_NEW
+                        Constants::ORDER_STATUS_PENDING
+                    ]
+                ],
+                'defaultSort' => ['dateOrdered', 'desc']
+            ],
```

Comment:
@shnsumit, should we switch the order of this?

<img width="271" alt="image" src="https://user-images.githubusercontent.com/11489608/147120659-798183f2-67fd-4d25-a3bf-03155a361f7b.png">